app/src/main/java/org/fossasia/openevent/general/event/types/EventType.kt

```diff
@@ -11,7 +11,7 @@ import com.github.jasminb.jsonapi.annotations.Type
 data class EventType(
     @Id(LongIdHandler::class)
     @PrimaryKey
-    val id: Long?,
-    val name: String?,
-    val slug: String?
+    val id: Long,
+    val name: String,
+    val slug: String
```

Comment:
why? These values will never be null?

Reply:
will fix them in #1584

Reply:
yes JSON-API id's can never be null and neither can name or slug